src/tests/ftest/rebuild/basic.yaml

```diff
@@ -9,6 +9,9 @@ server_config:
     targets: 2
     nr_xs_helpers: 0
     scm_size: 4
+    log_mask: DEBUG,MEM=ERR
```

Comment:
LGTM especially with a fairly minimal DD_MASK setting. It may be worth running the test (with Test-tags to include both pr and these tests test_simple_rebiuld, test_multipool_rebuild) with and without the log_mask change to compare typical sizes of artifacts to make sure there may not be some unexpected consequences in the CI environment.

Reply:
Thanks, Ken. I missed the test-tag; I've added it in the new merge commit. Let's wait for the CI...